src/test/resources/legacy/securityconfig_v6/whitelisting_settings.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Please add new lines to the end of all your .yml files.

Reply:
Yes, will do this

Reply:
Fixed.